### PR TITLE
Feature/save articles

### LINKED ISSUE #72: As a user, I can save/unsave articles and view saved articles in a separate section

### PR DESCRIPTION
**Pull request by anthony-iacono**
### Features
- Closes #72 

### Refactors 
- Point both History and SavedArticles to Feed.css for styling

### Issues and Technical Debt
- None

### Other Comments, Concerns, or Questions
- None

### Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -30,9 +30,12 @@ const Card = ({ title, image, id, shortUrl, sentiment, topic, updateUserSentimen
       >
         <p className="topic-text">{topic.toUpperCase()}</p>
         <h2 className="article-title cy-article-title">{title}</h2>
-        <img className="article-image cy-article-image" src={image} alt={title} />        
+        <img className="article-image cy-article-image" src={image} alt={title} />
       </a>
       <p className="abstract-text"> {abstract} </p>
+      <button onClick={ () => toggleSaved(id) } type="button">
```
Comment (nzambonivergara) [inline src/components/Card/Card.tsx:36]: This button should change its content when the user saves the article. Maybe we can have a method that checks whether the the article is in the savedArticles array and conditionally renders the button depending on whether it is already saved or not.

File: src/components/TopicForm/TopicForm.tsx [hunk 2]
```diff
@@ -24,6 +24,12 @@ const sectionTopics = availableTopics.map(topic => {
   return (
     <>
       <Header />
+      <Link to="/saved">
```
Comment (nzambonivergara) [inline src/components/TopicForm/TopicForm.tsx:27]: These buttons need styling. When view them on mobile view, they are too close together and it is difficult to click them.

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (nzambonivergara) [reply src/components/Card/Card.tsx:36]: Also, could we add styling to this button. You can use [this icon](https://fonts.google.com/icons?selected=Material+Icons+Outlined:favorite_border:) to save and [this one](https://fonts.google.com/icons?selected=Material+Icons+Outlined:favorite:) to remove from saved.

**Review by nzambonivergara — CHANGES_REQUESTED** [review]
Good job implementing this. I left comments about conditionally rendering the button to say 'Save article' and 'Remove from saved' depending on whether the article has been saved... I also put some icon suggestions if we want to skip the text.

File: src/components/TopicForm/TopicForm.tsx (hunk 2 above)
Comment (anthony-iacono) [reply src/components/TopicForm/TopicForm.tsx:27]: Fixed the margins, but still needs some pretty styling.

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (anthony-iacono) [reply src/components/Card/Card.tsx]: Fixed!

**Review by najinl — APPROVED** [review]

**Pull request merged**